beets/importer.py

```diff
@@ -1385,6 +1412,11 @@ def apply_choice(session, task):
 
     task.add(session.lib)
 
+    # If ``set_fields`` is set, set those fields to the
+    # configured values.
+    if config['import']['set_fields']:
+        task.set_fields()
```

Comment:
I recommend moving this stanza above the call to `task.add`. This way, we'll modify the values on the Album before ever putting them in the database. This will be marginally more efficient and, more importantly, will be a little more predictable.